parsers/mainParser.js

```diff
@@ -0,0 +1,19 @@
+const { confirmTransfer } = require('utilities/operations');
```

Comment:
it is advisable to process transactions not in operations, but to create files in the parsers folder, so that in case of adding logic, it would be easier to implement there

Reply:
+